skyportal/handlers/api/image_analysis.py

```diff
@@ -0,0 +1,451 @@
+
+        # ## Astrometric refinement
+        # Refining the astrometric solution based on the positions of detected objects and catalogue stars with scamp
+        wcs = pipeline.refine_astrometry(
+            obj, cat, wcs=wcs, method='scamp', cat_col_mag='rmag', verbose=True
+        )
+        if wcs is not None:
+            # Update WCS info in the header
+            astrometry.clear_wcs(
+                header,
+                remove_comments=True,
+                remove_underscored=True,
+                remove_history=True,
+            )
+            header.update(wcs.to_header(relax=True))
+
+        # ## Photometric calibration
+
+        # Positionally matching detected objects with catalogue stars
+        # Then building the photometric model for their instrumental magnitudes
+
+        # Photometric calibration using 2 arcsec matching radius, r magnitude, g-r color and second order spatial variations
+        m = pipeline.calibrate_photometry(
+            obj,
+            cat,
+            sr=2 / 3600,
+            cat_col_mag='rmag',
+            cat_col_mag1='gmag',
+            cat_col_mag2='rmag',
+            max_intrinsic_rms=0.02,
+            order=2,
+            verbose=True,
+        )
+
+        # The code above automatically augments the object list with calibrated magnitudes, but we may also do it manually
+        # obj['mag_calib'] = obj['mag'] + m['zero_fn'](obj['x'], obj['y'])
+        # obj['mag_calib_err'] = np.hypot(obj['magerr'], m['zero_fn'](obj['x'], obj['y'], get_err=True))
+
+        # ## Simple catalogue-based transient detection
+        # Some transients may already be detected by comparing the detected objects with catalogue BUT limited approach
+
+        # Filtering of transient candidates
+        candidates = pipeline.filter_transient_candidates(
+            obj, cat=cat, sr=2 / 3600, verbose=True
+        )
+        log(f'{candidates}')
+
+        # Creating cutouts for these candidates and vizualizing them (6238 ???)
+        filtered = []
+        for i, cand in enumerate(candidates):
+            dist = spherical_distance(
+                candidates[i]['ra'], candidates[i]['dec'], 191.4734, 89.1846
+            )  # XRT enhanced position
+            if dist < 4 / 3600:
+                filtered.append(candidates[i])
+
+        for i, cand in enumerate(filtered):
+            # Create the cutout from image based on the candidate
+            cutout = cutouts.get_cutout(image, cand, 20, mask=mask, header=header)
+            # We may directly download the template image for this cutout from HiPS server - same scale and orientation
+            cutout['template'] = templates.get_hips_image(
+                'PanSTARRS/DR1/r', header=cutout['header']
+            )[0]
+            # We do not have difference image, so it will only display original one, template and mask
+            plots.plot_cutout(cutout, qq=[0.5, 99.9], stretch='linear')
+
```

Comment:
What is happening here? What is 6238? Why are there hard-coded coordinates for the spherical distance? 
What is XRT enhanced position?

Reply:
@mcoughlin

Reply:
@Theodlz 191.4734, 89.1846 should instead be obj.ra.deg, obj.dec.deg and can remove the XRT comment,